### DISCUSSION TITLE
Slicer Thumbnail is missing...

### DISCUSSION THREAD

**Discussion by triton-nereide** [UnchartedBull/OctoDash#3742] [Q&A] [answered]
I had an issue with octoprint first:

when I first upload a file from prusa slicer, the thumnails show up correctly, after refreshing the page in firefox / edge they are gone. If I try to say "load image again", nothing changes. If I try to "open image in new tab" it shows the following location in the url: https://192.168.0.222/downloads/files/local/injector_0.2mm_PETG_MK3S_3h33m.aw.png?20230703083800
when I try to find the image via ssh (putty), it tells me, that the directories don´t exist. The command "ls" does not show the folders.
After the print, the thumnails in print job history do not show up.

I solved the problem by following a simple direction from octoprint forums: try a fresh install of Slicer Thumbs - I tried, and had to do this 2 times, the second time I created the missing directories (downloads/files/local) before the reinstall. Everything went O.K., now the thumbs show up in Octoprint, even after refreshing the page in firefox and edge.

But now I have a new issue: The thumbnail inside the progress circle in octodash went missing. Can you please tell me from where octodash is taking the slicer thumb to display it? Is there a possibility to solve this?

My system: Octoprint 1.9.1, Octopi 1.0, Octodash 2.3.1 -> all up to date in a fresh install from 3 days ago, same for all plugins...
it looks like "slicer thumbs" and "octodash" are cannibalizing themselves?

best regards and thank you for helping if possible,
C

**Comment by jneilliii** [comment, marked as answer]
Disable the option in Slicer Thumbnails to store the file in the uploads directory and re-upload your gcode files.

**Reply by triton-nereide** [reply]
O.K. thanx, I already changed the setting, but I have to wait until the current print is finished to try out, if that will work. For now the images are showingh inside octoprint at least, for octodash I have - as I said - to wait. Do You - by any chance - know, why - even with my previous changes - no images show up in printjob history? Will they - after changing to the proposed setting?

**Reply by jneilliii** [reply]
I don't use PrintJobHistory, so can't say for sure.
